docs/layouts/_default/baseof.html

```diff
@@ -38,7 +38,7 @@
   </head>
   <body class="dark:bg-dark font-sans antialiased flex flex-col min-h-full{{ if $hasBottomNav }} pb-16 lg:pb-0{{ end }}" onload="showBody()" style="visibility: hidden;">
     <div class="flex-1 z-1">
-      {{ partial "banner.html" . }}
+      {{ /* {{ partial "banner.html" . }} */}}
```

Comment:
There's an extra space. It needs to be:

```
{{/* {{ partial "banner.html" . }} */}}
```

Reply:
Ugh, sorry, I should have tested it out. I fixed the spacing and actually ran it locally this time 😄

Reply:
Do you love gotmpl yet?

Reply:
Go templates are actually quite nice for HTML, though a bit counter-intuitive at first. For YAML, on the other hand...

Reply:
Hah, I'm just glad I get to share some of the misery that's been isolated to the helm charts ;)